tests/Sinch.Tests/e2e/Verification/VerificationStatusTests.cs

```diff
@@ -11,20 +11,20 @@ public class VerificationStatusTests : VerificationTestBase
         private readonly ReportSmsVerificationResponse _smsVerificationReportResponse =
             new ReportSmsVerificationResponse()
             {
-                Id = "1234567890",
+                Id = "some_string_value",
                 Method = VerificationMethod.Sms,
                 Price = new PriceBase
                 {
                     VerificationPrice = new()
                     {
-                        CurrencyId = "USD",
-                        Amount = 0.0127
+                        CurrencyId = "some_string_value",
+                        Amount = 1.1
                     }
                 },
-                Reason = Reason.Fraud,
-                Reference = "12345",
-                Source = Source.Intercepted,
-                Status = VerificationStatus.Fail
+                Reason = new Reason("some_string_value"),
```

Comment:
Why updating all the fields with values different from the enums defined in the documentation or not meaningful ?
It would be better to update the mocks instead

Reply:
It's not a mock, it oas file, and need to update example which can take some time